app/Http/Controllers/Plan/PlansController.php

```diff
@@ -947,6 +947,15 @@ public function translate(Request $request, $plan_id, $user = null, $compare_pro
         
         $plan = $this->plan_service->translate($plan_id, $bible, $user_id, $draft, $show_details);
 
+        if ($show_details === true) {
```

Comment:
since show_details is sent to plan_service->translate, it seems better for setPlaylistItemsForEachPlaylist to be part of translate, and not here